src/Servers/IIS/IIS/test/IIS.Shared.FunctionalTests/ApplicationInitializationTests.cs

```diff
@@ -84,6 +102,7 @@ public async Task ApplicationInitializationPageIsRequested(HostingModel hostingM
             await Helpers.Retry(async () => await File.ReadAllTextAsync(Path.Combine(result.ContentRoot, "Started.txt")), TimeoutExtensions.DefaultTimeoutValue);
             StopServer();
             EventLogHelpers.VerifyEventLogEvent(result, EventLogHelpers.Started(result), Logger);
+            EventLogHelpers.VerifyEventLogEvent(result, EventLogHelpers.ShutdownMessage(result), Logger);
```

Comment:
```suggestion
```

Could add test variation to enable shutdown delay to keep this, but since the backport change isn't about this case I don't care.